components/eamxx/scripts/atmchange

```diff
@@ -17,10 +17,17 @@ from atm_manip import get_xml_node, atm_config_chg_impl
 from utils import run_cmd_no_fail, expect
 
 ###############################################################################
-def atm_config_chg(changes, no_buffer=False):
+def atm_config_chg(changes, no_buffer=False, reset=False):
 ###############################################################################
     expect(os.path.exists("namelist_scream.xml"),
-           "No pwd/namelist_scream.xml file is present. Please rum from a case dir that has been setup")
+           "No pwd/namelist_scream.xml file is present. Please run from a case dir that has been set up")
+
+    if reset:
+        run_cmd_no_fail(f"./xmlchange SCREAM_ATMCHANGE_BUFFER=''")
+        print("All buffered atmchanges have been removed. A fresh namelist_scream.xml will be generated the next time buildnml (case.setup) is run.")
```

Comment:
Is this true? Don't you need to rm namelist_scream.xml in order to get a new one generated by buildnml?

Reply:
Nah, no need to re-rerun case.setup. I forgot about your recent change that XML is _always_ regenerated.